scripts/ci/check_compliance.py

```diff
@@ -133,6 +169,8 @@ class ComplianceTest:
       statement runs). That avoids swallowing errors, because main() reports
       them to GitHub
     """
+    path_hint = "<git-top>"
```

Comment:
The pydoc comment above now no longer is valid:

```
    Base class for tests. Inheriting classes should have a run() method and set
    these class variables:
```

Reply:
Sure - should I just remove the comment?

Reply:
I'd add the default value to the comment, indicating that it can be overridden?

Reply:
Fixed!